src/rttui/app.rs

```diff
@@ -324,6 +368,7 @@ impl App {
                             let extension = match tab.format() {
                                 DataFormat::String => "txt",
                                 DataFormat::BinaryLE => "dat",
+                                DataFormat::Defmt => panic!("can not write Defmt output to disk"),
```

Comment:
```suggestion
                                DataFormat::Defmt => panic!("Cannot write defmt output to disk."),
```

Reply:
Please do not panic here. It is not nice if the application crashes because of a missconfiguration.